### ISSUE TITLE
[nuxtjs] vue-screen breaks on ios 12,13

### ISSUE THREAD

**Issue by regenrek** [reegodev/vue-screen#30]
Hi, 

I stumpled upon some issues with the vue-screen library on ipads. The client side js breaks
when I've added the nuxtjs vue-screen library on ios 12 and 13. 

IOS 14 Works fine (Tested with Browserstack)
vue-screen version: 1.5.2

```
TypeError: _query.addEventListener is not a function. (In '_query.addEventListener('change', function (e) {(…)promiseReactionJob @ (program):1
```

**Comment by reegodev** [comment]
Hi @regenrek , i can see why. Thanks for reporting!
Gonna fix it asap!

**Comment by reegodev** [comment]
Fixed in version 1.5.3
